src/clld/web/templates/datatable.mako

```diff
@@ -1,8 +1,12 @@
 <table id="${obj.eid}" cellpadding="0" cellspacing="0" border="0" class="table table-bordered order-column compact stripe">
     <thead>
         <tr>
-            % for col in obj.cols:
-            <th>${col.js_args['sTitle']}</th>
+            % for col in ctx.cols:
```

Comment:
The datatable object is passed into the template as `obj`. Using the global `ctx` may work in some cases, but not in general!